lib/rules/utils/ast-utils.js

```diff
@@ -616,10 +616,15 @@ module.exports = {
      *
      * @param {ASTNode} node - A function node to check.
      * @param {SourceCode} sourceCode - A SourceCode instance to get comments.
+     * @param {boolean} [capIsConstructor = true] - `false` disables the assumption that functions which name starts
```

Comment:
Do we want to update the pseudo code above in this comment to account for the new function signature?

Reply:
It's done now :)